backend/fleet_management/management/commands/populate_database.py

```diff
@@ -19,31 +19,49 @@ class Command(BaseCommand):
     help = "Populates database with fake items."
 
     def handle(self, *args, **options):
-        """Populate database with fake objects."""
+        superuser = User()
+        superuser.is_active = True
+        superuser.is_superuser = True
+        superuser.is_staff = True
+        superuser.username = 'hello@codeforpoznan.pl'
+        superuser.email = 'hello@codeforpoznan.pl'
+        superuser.set_password('cfp123')
+        superuser.country = 'UA'
+        superuser.save()
 
-        self.stdout.write(self.style.SUCCESS('Creating 30 cars'))
-        for _ in tqdm(range(30)):
+        superuser = User()
+        superuser.is_active = True
+        superuser.is_superuser = False
+        superuser.is_staff = False
+        superuser.username = 'ola@pah.org.pl'
+        superuser.email = 'ola@pah.org.pl'
+        superuser.set_password('pah123')
+        superuser.country = 'UA'
+        superuser.save()
+
+        self.stdout.write(self.style.SUCCESS('Creating 5 cars'))
+        for _ in tqdm(range(5)):
             CarFactory.create()
 
-        self.stdout.write(self.style.SUCCESS('Creating 50 users'))
+        self.stdout.write(self.style.SUCCESS('Creating 5 users'))
```

Comment:
Do you have a reason to reduce the number of generated instances?

Reply:
Yes - in development having 50 users is not required. 
Having these high numbers makes seeding longer. 
Also - the page loads longer on each refresh during development. 

High numbers in seeds are more of an edge case scenario for testers rather than good development numbers for developers.